src/main/kotlin/racingcar/Application.kt

```diff
@@ -1,5 +1,97 @@
 package racingcar
 
+import camp.nextstep.edu.missionutils.Console
+import camp.nextstep.edu.missionutils.Randoms
+
 fun main() {
-    // TODO: 프로그램 구현
+    println("경주할 자동차 이름을 입력하세요.(이름은 쉼표(,) 기준으로 구분)")
+    val inputCar = Console.readLine()
+    var cars = inputCar.split(",")
+    isValidCarName(cars)
+    cars = duplicateName(cars)
+
+    println("시도할 횟수는 몇 회인가요?")
+    val inputMove = Console.readLine()
+    val move = isMovementValid(inputMove)
+    race(cars, move)
+}
+
+fun isValidCarName(cars: List<String>){
```

Comment:
'is'로 시작하는 함수는 일반적으로 boolean 값을 반환할때 명명한다고 합니다
cars라는 리스트를 인자로 받고 있기 때문에 함수명도 복수로 짓는게 좋을 것 같네요!

```suggestion
fun validateCarNames(cars: List<String>){
```
만약 제가 명명한다면 이렇게 이름을 지을것 같습니다!

Reply:
맞아요! 'is뭐시기?' 라고 물어보면 '예스 or 노' 로 대답하고 싶은 심리가 생기네요
또한 복수의 인풋을 받는것도 고려해야 하는점은 미처 몰랐습니다